index.js

```diff
@@ -197,10 +197,11 @@ BabelClient.prototype.getAnnotations = function(token, querystringMap, callback)
  * @param {string} data.annotatedBy
  * @param {string} data.motiviatedBy
  * @param {string} data.annotatedAt
- *
- * @callback callback
+ * @param {object} options that control the request being made to babel.
+ * @param {boolean} options.xIngestSynchronously
+ * @param callback
  */
-BabelClient.prototype.createAnnotation = function(token, data, callback){
+BabelClient.prototype.createAnnotation = function(token, data, options, callback){
```

Comment:
yeah that would work, there's really no nice way of defaulting parameters in javascript (at least not ecmascript 6 is implemented  across the board: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/default_parameters)